content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,155 @@
+**Notes**: the only supported values are `tracecontext`, `b3`, `b3multi`, `none`, `datadog` along with `xray` for the `java` sdk<br>
+Values MUST be deduplicated in order to register a `Propagator` only once<br>
+
+`OTEL_TRACES_SAMPLER & OTEL_TRACES_SAMPLER_ARG`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_SAMPLE_RATE`<br>
+`OTEL_TRACES_SAMPLER`: Sampler to be used for traces & `OTEL_TRACES_SAMPLER_ARG`: String value to be used as the sampler argument<br>
+**Notes**: The specified value will only be used if `OTEL_TRACES_SAMPLER` is set. Each Sampler type defines its own expected input, if any. Invalid or unrecognized input MUST be logged and MUST be otherwise ignored, i.e. the implementation MUST behave as if `OTEL_TRACES_SAMPLER_ARG` is not set<br>
+Mapped values between `OTEL_TRACES_SAMPLER` & `DD_TRACE_SAMPLE_RATE`:<br>
+  - `parentbased_always_on`|`1.0`
+  - `parentbased_always_off`|`0.0`
+  - `parentbased_traceidratio`|`${OTEL_TRACES_SAMPLER_ARG}`
+  - `always_on`|`1.0`
+  - `always_off`|`0.0`
+  - `traceidratio`|`${OTEL_TRACES_SAMPLER_ARG}`
+
+`OTEL_TRACES_EXPORTER`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_ENABLED=false` <br>
+Trace exporter to be used<br>
+**Notes**: only a value of `none ` is accepted<br>
+
+`OTEL_METRICS_EXPORTER`
+: **Corresponding Datadog Environment Variable**: `DD_RUNTIME_METRICS_ENABLED=false` <br>
+Metrics exporter to be used<br>
+**Notes**: only a value of `none` is accepted<br>
+**Not Supported In**: `php` sdk<br>
+
+`OTEL_RESOURCE_ATTRIBUTES`
+: **Corresponding Datadog Environment Variable**: `DD_TAGS` <br>
+Key-value pairs to be used as resource attributes. See [Resource semantic conventions][11] for details<br>
+**Notes**: `deployment.environment` maps to the `DD_ENV` environment variable<br>
+`service.name` maps to the `DD_SERVICE` environment variable<br>
+`service.version` maps to the `DD_VERSION` environment variable
+
+`OTEL_SDK_DISABLED`
+: **Corresponding Datadog Environment Variable**: `!DD_TRACE_OTEL_ENABLED` <br>
+Disable the SDK for all signals<br>
+**Notes**: Mapped values between `OTEL_SDK_DISABLED` & `DD_TRACE_OTEL_ENABLED`:<br>
+  - `true`|`false`
+  - `false`|`true`
+**Not Supported In**: `ruby` & `golang` sdk's<br>
+
+[for more info, see the OpenTelemetry documentation][10]
+## Java Specific Configuration
+
+`OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED`
+: **Corresponding Datadog Environment Variable**: `!DD_INTEGRATIONS_ENABLED` <br>
+Set to `false` to disable all instrumentation in the agent<br>
+**Notes**: Mapped values between `OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED` & `DD_INTEGRATIONS_ENABLED`:<br>
+  - `true`|`false`
+  - `false`|`true`
+
+`OTEL_INSTRUMENTATION_[NAME]_ENABLED`
+: **Description**: Enables/disables the named OTel drop-in instrumentation<br>
+
+`OTEL_JAVAAGENT_CONFIGURATION_FILE`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_CONFIG` <br>
+Path to valid Java properties file which contains the agent configuration<br>
```

Comment:
Note: when`OTEL_JAVAAGENT_CONFIGURATION_FILE` and `DD_TRACE_CONFIG` are both set we apply the settings from both files.